scripts/buildkite/release/generate-trigger.sh

```diff
@@ -14,6 +14,7 @@ steps:
     depends_on: add-release-commits
     key: main-pipeline-build
     label: Trigger the main pipeline
+    timeout: 1080
```

Comment:
```suggestion
```

According to the Buildkite help, `trigger` steps to not seem to support a `timeout` field. 🤔 

https://buildkite.com/docs/pipelines/trigger-step#trigger-step-attributes

Reply:
ohh, I assume then it's not accounted directly. Thanks for checking